kotlin-android/app/src/main/java/com/mongodb/tasktracker/model/Task.kt

```diff
@@ -5,19 +5,15 @@ import io.realm.annotations.PrimaryKey
 import io.realm.annotations.Required
 import org.bson.types.ObjectId
 
-enum class TaskStatus(val displayName: String) {
-    Open("Open"),
-    InProgress("In Progress"),
-    Complete("Complete"),
-}
```

Comment:
I still kinda disagree with this change... every additional file is a chance for confusion

Reply:
I'm sure we'll revise it in the future -- if this ends up being confusing for even one user, I'll change it. Deal?